### PR TITLE
Travis added

### PR DESCRIPTION
**Pull request by ppamorim**
Let's travis do the test for you.

### CODE REVIEW

**Comment by zhanghai** [comment]
The language level of this project should remain 1.7 to be compatible with Android; and it would be better if these commits could be squashed; and where did that checkstyle.xml came from?

**Comment by ppamorim** [comment]
I don't have changed the version of Java, review the changes.
https://github.com/DreaminginCodeZH/MaterialProgressBar/pull/14/files

The checkedstyle.xml will be used by Travis CI.

**Comment by PaulWoitaschek** [comment]
+1 Testing is always a good idea

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  components:
+    - build-tools-23.0.2
+    - android-23
+    - extra-android-m2repository
+
+  licenses:
+    - android-sdk-license-.+
+
+script:
+  ./gradlew checkstyle build
```
Comment (webmaster128) [inline .travis.yml:17]: Please add a "\n" at the end of the file because a line is always followed by a linebreak

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+language: android
+
+before_install:
+    - echo yes | android update sdk --filter extra-android-support --no-ui --force > /dev/null
+    - echo yes | android update sdk --filter extra-android-m2repository --no-ui --force > /dev/null
```
Comment (webmaster128) [inline .travis.yml:5]: whitespace consistency: 4 or 2 spaces per indent level?

**Comment by webmaster128** [comment]
LGTM. It is good to have a basic testing framework set up that can be adjusted over time.

**Pull request closed without merging**